src/python/system/environment.py

```diff
@@ -308,7 +313,7 @@ def get_environment_settings_as_string():
       sanitizer_options_file_path = (
           android.device.get_sanitizer_options_file_path('ASAN'))
       environment_string += (
-          '[Environment] ASAN Options file = %s with contents %s \n' %
+          '[Environment] ASAN options file "%s" with contents %s \n' %
```

Comment:
change "contents %s \n" to "contents:\n%s\n" to match others.

Reply:
Done